src/hp/derp/derp.rs

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
drope these please, we don’t have it any of the files and instead add a comment what file this is based on